src/home/Home.jsx

```diff
@@ -19,6 +20,7 @@ export default function Home() {
                 </div>
             </section>
             <GetInTouch />
+            <FAQ />
```

Comment:
Also, `FAQ` is above `GET in Touch` section as per the Figma file 😉

Reply:
Hi @mtreacy002  , can you please tell me the correct order of these components :)

Reply:
@naveen8801 , you can refer to the Figma file in issue #213 description 😉